addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -104,6 +104,10 @@ def format_float(self, amount, precision_digits):
             return None
         return float_repr(float_round(amount, precision_digits), precision_digits)
 
+    def _get_currency_decimal_places(self, record):
```

Comment:
Actually, it could be any type of record that has a `currency_id` field set on it.
It could be an invoice, an invoice_line, a sale_order, etc

I'm not sure to really understand your comment:
> The purpose of this model is to be abstract of the business model

Well... The method is abstracted accordingly.
Considering that, could you tell me why we would need to prefix it with `invoice` ?

Reply:
What do you think of that:
```py
def _get_currency_values(self, currency):
    return {
        'currency': currency,
        'currency_name': currency.name,
        'currency_dp': currency.decimal_places,
    }
```
That way, it looks more generic to me. You just need to put the results in all dictionaries